plutus-scb/src/Plutus/SCB/ContractCLI.hs

```diff
@@ -62,24 +62,36 @@ updateParser =
         (fullDesc <>
          progDesc "Update the contract. The request must be supplied on stdin.")
 
-runCliCommand ::
+
+exportSignatureParser :: Mod CommandFields Command
+exportSignatureParser =
+    command "export-signature" $
+    info (pure ExportSignature) (fullDesc <> progDesc "Export the contract's signature.")
+
+runCliCommand :: forall s m.
        ( AllUniqueLabels (Input s)
        , AllUniqueLabels (Output s)
        , Forall (Input s) FromJSON
+       , Forall (Input s) ToJSON
+       , Forall (Output s) ToJSON
        , Forall (Output s) Monoid
        , Forall (Output s) Semigroup
+       , EndpointToSchema (s .\\ BlockchainActions)
        , MonadIO m
        )
     => Contract s Text ()
     -> Command
-    -> m (Either (ResumableError Text) (Response s))
-runCliCommand schema Initialise = pure $ initialResponse schema
+    -> m (Either BS8.ByteString BS8.ByteString)
```

Comment:
using bytestrings seems a bit...bad form. Also, isn't the `Left` case pretty printed twice, once here and once in `commandLineApp`?